samples/modules/lvgl/demos/README.rst

```diff
@@ -18,15 +18,27 @@ A sample showcasing upstream LVGL demos.
 * Widgets
       Shows how the widgets look like out of the box using the built-in material theme.
 
+More details can be found in `LVGL demos Readme`_.
+
 Requirements
 ************
 
 * A board with display, ideally with 480x272 resolution or higher.
+* A pointer input device: touchpad, mouse, or touch screen capable display, compatible with :dtcompatible:`zephyr,lvgl-pointer-input`.
+
+Note that other input devices types are not demonstrated in these demos, namely keyboards, keypads (:dtcompatible:`zephyr,lvgl-button-input`), rotary encoders (:dtcompatible:`zephyr,lvgl-encoder-input`) and hardware buttons (:dtcompatible:`zephyr,lvgl-button-input`).
```

Comment:
The first compatible should be keypad-input :^).
Also keep in mind that this needs to be adjusted once [https://github.com/zephyrproject-rtos/zephyr/pull/68358](https://github.com/zephyrproject-rtos/zephyr/pull/68358) is merged.